src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css

```
@@ -863,3 +863,24 @@ table.striped > tbody > tr > th {
         padding-right: 12px;
     }
 }
+
+pre.snippet {
+    background-color: #e8e8e8;
+    padding: 10px;
+    margin: 12px 0;
+    overflow: auto;
+    white-space: pre;
+}
+
+pre.snippet .italic {
+    font-style: italic;
+}
+
+pre.snippet .bold {
+    font-weight: bold;
+}
+
+pre.snippet .highlighted {
+    background-color: sandybrown;
```

Here I think the color clashes a bit with the very similar orange we use for table headers/tabs. I'm not really sure about this one, but maybe a "watered down" orange with a similar tone but less saturation could work. 
```suggestion
    background-color: #f7c590;
```